src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -1992,4 +2012,92 @@ export class PatientViewPageStore {
     @computed get genomeNexusInternalClient() {
         return new GenomeNexusAPIInternal(this.referenceGenomeBuild);
     }
+
+    @computed get sampleMsiHInfo() {
+        return getSampleClinicalDataMapByThreshold(
+            this.clinicalDataForSamples.result,
+            CLINICAL_ATTRIBUTE_ID_ENUM.MSI_SCORE,
+            MSI_H_THRESHOLD
+        );
+    }
+
+    @computed get sampleTmbHInfo() {
+        return getSampleClinicalDataMapByThreshold(
+            this.clinicalDataForSamples.result,
+            CLINICAL_ATTRIBUTE_ID_ENUM.TMB_SCORE,
+            TMB_H_THRESHOLD
+        );
+    }
+
+    @computed get otherBiomarkerQueries() {
+        const queries: AnnotateMutationByProteinChangeQuery[] = [];
+        if (_.keys(this.sampleMsiHInfo).length > 0) {
```

Comment:
The `if` blocks are almost identical, may be worth combining them.

Reply:
Yeah, previously I don't like to add too many key value pairs for these. But since I'm gonna combine the logic in the PatientViewPage anyway, I will update here too.